styles/grid.css

```diff
@@ -0,0 +1,19 @@
+@import url('./main.css');
+.grid-item {
+    background: rebeccapurple;
```

Comment:
use variables or hex code for colors, as different browsers interpret colors differently